requirements.txt

```diff
@@ -6,3 +6,4 @@ matplotlib
 numpy
 requests
 shapely
+descartes
```

Comment:
for future reference, please try to add new libraries in alphabetical order. It makes it easier to see if the one you want is here as the list grows.